apps/src/storage/dataBrowser/data-styles.module.scss

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file Styles shared by various react components in the applab data browser.
+ */
+
+@import 'color.scss';
+
+$rowHeight: 45px;
+$cellPadding: 10px;
+$editButtonCellWidth: 142px;
+$maxCellWidth: 350px;
+
+.row {
+    height: $rowHeight;
+}
+
+.cell {
+    padding: $cellPadding;
+    border: 1px solid gray;
+    font-size: 14px;
+    max-width: $maxCellWidth;
+    overflow: hidden;
+    &AddButton {
+        background-color: $lightest_teal
+    }
+    &EditButton {
+        background-color: $lightest_teal;
+        width: $editButtonCellWidth
```

Comment:
nit: add `;` -- there are a few places we need semicolons, so do a quick read through this file for those!

Reply:
Gah - I thought I got them all! Will scan and check again.

Reply:
I think/hope I got them all: https://github.com/code-dot-org/code-dot-org/pull/47567/commits/845a01ebbe30a19f206443cbefc520b383239e4c

Do you know if there is a plugin or something I could be using to catch these automatically?

Reply:
yeah, this will be solved with linting, which meg will be working on adding when she's back from being OOO